### DISCUSSION TITLE
View GCODE Model on Speeder Pad

### DISCUSSION THREAD

**Discussion by RobbieRw** [Guilouz/Klipper-Flsun-Speeder-Pad#47] [General]
Hi guys

Is there a way to see the gcode model im printing on the speeder pad? At the moment i just have a box with where i believe the preview should be. 

I have seen pictures of others with the preview. any help would be great. 

Thanks

**Comment by phnxdrgn536** [comment]
If you are using cure I think all you have to do is  save it as a .ufp and
it automatically includes the preview image.

**Comment by RobbieRw** [comment]
Hi

Oh really! I will give this a go.

Thanks

**Comment by RobbieRw** [comment]
Hi Joseph

Do you have a good cura profile for the v400? im getting slight stringing
with my models with a lot of retraction.

Thanks

Rob
